### PR TITLE
Add click listeners and fix tab index

### PR DESCRIPTION
**Pull request by OlliTietavainenVaadin**
* Added possibility to add click listeners to Stepper fields
* Fixed tab index behavior

### CODE REVIEW

File: addon/src/main/java/org/vaadin/risto/stepper/AbstractStepper.java [hunk 1]
```diff
@@ -192,6 +210,10 @@ public void setIncreaseIcon(Resource icon) {
         markAsDirty();
     }
 
+    public Registration addClickListener(ClickListener listener) {
```
Comment (ripla) [inline addon/src/main/java/org/vaadin/risto/stepper/AbstractStepper.java]: Javadoc

File: addon/src/main/java/org/vaadin/risto/stepper/AbstractStepper.java [hunk 2]
```diff
@@ -259,4 +281,12 @@ protected abstract T parseStringValue(String value)
      * @return the type of the value in this field
      */
     protected abstract Class<T> getValueType();
+
+    @FunctionalInterface
+    public interface ClickListener extends Serializable {
+        Method BUTTON_CLICK_METHOD = ReflectTools.findMethod(ClickListener.class,
```
Comment (ripla) [inline addon/src/main/java/org/vaadin/risto/stepper/AbstractStepper.java]: I would prefer a separate classes from Button. Even if they would be mostly copy-paste.

File: addon/src/main/java/org/vaadin/risto/stepper/client/shared/ClickRpc.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package org.vaadin.risto.stepper.client.shared;
+
+import com.vaadin.shared.MouseEventDetails;
+import com.vaadin.shared.communication.ServerRpc;
+
+@FunctionalInterface
+public interface ClickRpc extends ServerRpc {
```
Comment (ripla) [inline addon/src/main/java/org/vaadin/risto/stepper/client/shared/ClickRpc.java]: Javadoc

File: demo/src/main/webapp/VAADIN/themes/demo/addons.scss [hunk 4]
```diff
@@ -1,17 +1,7 @@
...
-@mixin addons {
-	@include stepper;
-	@include stepper;
-}
-
+/* This file is automatically managed and will be overwritten from time to time. */
+/* Do not manually edit this file. */
+
+/* Import and include this mixin into your project theme to include the addon themes */
+@mixin addons {
```
Comment (ripla) [inline demo/src/main/webapp/VAADIN/themes/demo/addons.scss]: Does the demo still work without addon.scss import?

File: addon/src/main/java/org/vaadin/risto/stepper/AbstractStepper.java (hunk 1 above)
Comment (OlliTietavainenVaadin) [reply addon/src/main/java/org/vaadin/risto/stepper/AbstractStepper.java]: done

File: addon/src/main/java/org/vaadin/risto/stepper/AbstractStepper.java (hunk 2 above)
Comment (OlliTietavainenVaadin) [reply addon/src/main/java/org/vaadin/risto/stepper/AbstractStepper.java]: done

File: addon/src/main/java/org/vaadin/risto/stepper/client/shared/ClickRpc.java (hunk 3 above)
Comment (OlliTietavainenVaadin) [reply addon/src/main/java/org/vaadin/risto/stepper/client/shared/ClickRpc.java]: done

File: demo/src/main/webapp/VAADIN/themes/demo/addons.scss (hunk 4 above)
Comment (OlliTietavainenVaadin) [reply demo/src/main/webapp/VAADIN/themes/demo/addons.scss]: I guess? It was removed by Gradle, I didn't touch it myself. Reverted anyway.

**Pull request merged**